xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1397,6 +1378,18 @@ namespace VIDEO
         art.insert(std::make_pair("fanart", fanart));
     }
 
+    // add online art
+    std::vector<CScraperUrl::SUrlEntry>& scraperArt = pItem->GetVideoInfoTag()->m_strPictureURL.m_url;
+    for (auto i = scraperArt.begin(); i != scraperArt.end(); ++i)
```

Comment:
You could use a range-based for like this `for (const auto& a : pItem->GetVideoInfoTag()->m_strPictureURL.m_url)`